engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelHydrationValidation.kt

```diff
@@ -157,7 +157,23 @@ internal class NadelHydrationValidation(
             }
         }
 
-        return duplicatedArgumentsErrors + remoteArgErrors
+        val missingActorArgErrors = actorField.arguments.mapNotNull { actorArg ->
+            val hydrationArg = hydration.arguments.find { it.name.equals(actorArg.name) }
```

Comment:
Nit: use `==` in Kotlin instead of `.equals`

In Kotlin `==` invokes `.equals` and `===` does the identity check like `==` does in Java.